person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :name, :age, :parent_permission
+  attr_accessor :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] Please add an instance variable for id and you can make it a set of random numbers.

![image](https://user-images.githubusercontent.com/68796232/173563588-3b0801cd-c9fd-4078-85bd-19db56088b27.png)